spring-boot-autoconfigure/src/main/java/org/axonframework/springboot/util/DeadLetterQueueProviderConfigurerModule.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright (c) 2010-2023. Axon Framework
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.axonframework.springboot.util;
+
+import org.axonframework.config.Configuration;
+import org.axonframework.config.Configurer;
+import org.axonframework.config.ConfigurerModule;
+import org.axonframework.eventhandling.EventMessage;
+import org.axonframework.messaging.deadletter.SequencedDeadLetterQueue;
+import org.axonframework.springboot.EventProcessorProperties;
+
+import java.util.Map;
+import java.util.Optional;
+import java.util.function.Function;
+import javax.annotation.Nonnull;
+
+/**
+ * A {@link ConfigurerModule} implementation dedicated towards registering a {@link SequencedDeadLetterQueue} provider
+ * with the {@link org.axonframework.config.EventProcessingConfigurer}.
+ * <p>
+ * Does so through invoking the
+ * {@link org.axonframework.config.EventProcessingConfigurer#registerDeadLetterQueueProvider(Function)} operation,
+ * utilizing the given {@code deadLetterQueueProvider}. Only processing groups for which the
+ * {@link EventProcessorProperties.Dlq#isEnabled() dead-letter queue is enabled} will receive the provided dead-letter
+ * queue.
+ *
+ * @author Steven van Beelen
```

Comment:
Gerard lost his credits here from the copy

Reply:
Great catch! Added him back in.